### PR TITLE
Update ConfigMap name for CNV

### PR DESCRIPTION
**Pull request by smalleni**
Updates ConfigMap name after recent changes.

Fixes: #761

Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

File: roles/uperf/templates/workload_vm.yml.j2 [hunk 1]
```diff
@@ -90,7 +90,7 @@ spec:
           - bash /tmp/uperf_script/run_script.sh
     name: cloudinitdisk
   - configMap:
-      name: uperf-test-{{ trunc_uuid }}
+      name: uperf-test-{{ item }}-{{ trunc_uuid }}
```
Comment (mukrishn) [inline roles/uperf/templates/workload_vm.yml.j2:93]: @smalleni, I think this should be `uperf-test-{{ ansible_loop.index0 }}-{{ trunc_uuid }}`

**Comment by ebattat** [comment]
Hi @smalleni,

Your latest changes solved it:

```
➜  benchmark-operator git:(fix_cnv) ✗ oc get vmi -n benchmark-operator -w
NAME                                AGE     PHASE     IP            NODENAME   READY
uperf-client-10.130.1.21-45d11b0d   2m14s   Running   10.130.1.23   master-0   True
uperf-server-0-45d11b0d             2m35s   Running   10.130.1.21   master-0   True
```

When you plan to merge it ?

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**